python/lsst/cp/pipe/cpCertify.py

```diff
@@ -137,11 +137,6 @@ def convertStorageClass(self, data, datasetTypeName):
         data : `lsst.afw.image.Image` or `lsst.afw.image.MaskedImage`
             Converted image data to register.
         """
-        if datasetTypeName in ('bias', 'dark'):
-            data = data.getImage()
-        # elif datasetTypeName in ('flat', ):
-        #     data = data.getMaskedImage()
-
```

Comment:
You deleted this, and now the function just returns "data", which is the input. So, why do we need this function?

Reply:
Inertia.  Deleted.